projects/sb-user-management/src/lib/user-count/user-count.component.css

```diff
@@ -0,0 +1,17 @@
+h1{
+    color: green;text-align: center !important;
```

Comment:
@poojakarma 
Please don't use **! important** as it is considered as bad practice.

Reply:
Removed